packages/leancode_contracts_generator/lib/src/statements/dto_handler.dart

```diff
@@ -30,7 +30,9 @@ class DtoHandler extends StatementHandler {
       // of type parameters of this DTO. This means this will break
       // inheritance. Something like `class A extends B<T> {}`
       if (typeDescriptor.genericParameters.isNotEmpty) {
-        b.methods.clear();
+        b.methods.removeWhere(
+          (m) => m.name == 'toJson' || m.name == 'fromJson',
```

Comment:
fromJson is a factory constructor, not a method. And since it is a constructor, it does not have to be removed as it is not part of the interface of a type